ajax/libs/open-iconic/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "open-iconic",
+  "filename": "css/open-iconic.min.css",
+  "description": "An open source icon set with marks in SVG, sprite, webfont and raster format",
+  "version": "1.1.1",
+  "keywords": [
+    "icon",
+    "iconic",
+    "open-iconic",
+    "svg",
+    "sprite",
+    "font",
+    "png",
+    "webp"
+  ],
+  "homepage": "https://useiconic.com/open",
+  "author": {
+    "name": "Iconic",
+    "email": "yourfriends@useiconic.com",
+    "web": "http://useiconic.com/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/iconic/open-iconic.git"
+  },
+  "licenses": [
+    "MIT",
+    "OFL 1.1"
```

Comment:
@kennynaoh Please modify it to `"OFL-1.1"`, ref: https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json#L242